doc/thirdPartyRunPlugins.md

```diff
@@ -0,0 +1,35 @@
+# Third-Party Run Plugins
+
+Collection of third-party Run plugins created by the community that aren't distributed with PowerToys.
+
+## Installing / Upgrading
+
+The recommended path for the plugins is `%LOCALAPPDATA%\Microsoft\PowerToys\PowerToys Run\Plugins`.
```

Comment:
so if you have a machine wide installer ... this still works?  Doesn't machine wide install at program files?  TIL

Reply:
We have two folders: Program files and user profile. And I have in mind that there were some fixing PRs in the past.

Reply:
I confirm that.
Plugins are loaded from `INSTALL-PATH\PowerToys\modules\launcher\Plugins` + `%LOCALAPPDATA%\Microsoft\PowerToys\PowerToys Run\Plugins`.
I am using the second path with a machine installation since 0.68.0 but IIRC should work with older versions too.